sql/core/src/test/resources/sql-tests/inputs/except.sql

```diff
@@ -0,0 +1,57 @@
+-- Tests different scenarios of except operation
+create temporary view t1 as select * from values
+  ("one", 1),
+  ("two", 2),
+  ("three", 3),
+  ("one", NULL)
+  as t1(k, v);
+
+create temporary view t2 as select * from values
+  ("one", 1),
+  ("two", 22),
+  ("one", 5),
+  ("one", NULL),
+  (NULL, 5)
+  as t2(k, v);
+
+
+-- Except operation that will be replaced by left anti join
+SELECT * FROM t1 EXCEPT SELECT * FROM t2;
```

Comment:
To see whether these SQL use the rule you added, you can add `EXPLAIN EXTENDED` before the actual query `SELECT * FROM t1 EXCEPT SELECT * FROM t2`

You will see the optimized plan and check whether the rule is applicable.

Reply:
BTW, because our plan is not stable, I am not asking you for outputting the plan in the test cases. This is just to help you debug the codes. Thanks!

Reply:
Thanks, good to know. I have been manually testing it via spark shell.

Reply:
How about the result?

Reply:
All verified via spark shell using `createOrReplaceTempView`

Reply:
And please let me know when it is right time to squash the commits..